libweb3jsonrpc/Test.cpp

```diff
@@ -125,7 +155,8 @@ std::string Test::test_importRawBlock(string const& _blockRLP)
     catch (ImportBlockFailed const& e)
     {
         cwarn << diagnostic_information(e);
-        throw JsonRpcException("Block import failed.");
+        ImportResult const& res = *boost::get_error_info<dev::eth::errinfo_importResult>(e);
```

Comment:
So to fix the unit tests, just check here that the pointer returned from `get_error_info` is not null before dereferencing, don't add additional string if it's null.

Then this PR is fine, but I'm not sure if it helps you at all.